cosi/cosi_new.go

```diff
@@ -0,0 +1,130 @@
+package cosi
+
+import (
+	"crypto/cipher"
+
+	"github.com/dedis/kyber/abstract"
+)
+
+func Commit(suite abstract.Suite, s cipher.Stream) (abstract.Scalar, abstract.Point) {
+	// compute random scalar
+	// compute commitment
+	// return both
+}
+
+func AggregateCommitments(suite abstract.Suite, commitments []abstract.Point, mask *Mask) (abstract.Point, *Mask) {
```

Comment:
What do you do with the mask ? Do you assume that the length of the given commitments is the same as the total size of the participant set, with some nil entries for absent participants ?
Otherwise, I'm wondering if the `mask` is really belonging here. 
Another idea would be to make that function return the mask. Use a more structure-d approach like `Commiment struct { Index int, C abstract.Point}` so we know the index of each commitment. Then one just need to call 
`aggCommit, mask  = AggregateCommitments(suite, commitments, n)` with n being the length of the set of participants.
Just ideas... ;)